File: src/netcore-api/SIL.XForge.WebApi.Server/Models/Lexicon/LexConfigPictures.cs
namespace SIL.XForge.WebApi.Server.Models.Lexicon
{
    public class LexConfigPictures : LexConfigMultiText
    {
        public LexConfigPictures()
        {
            Type = Pictures;
            Label = "Pictures";
        }

        public string CaptionLabel { get; set; } = "Captions";
        public bool CaptionHideIfEmpty { get; set; } = true;
    }
}

File: src/Api/Model/Languageforge/Lexicon/Config/LexConfigPictures.php
<?php

namespace Api\Model\Languageforge\Lexicon\Config;

class LexConfigPictures extends LexConfigMultiText
{
    public function __construct()
    {
        parent::__construct();
        $this->type = self::PICTURES;
        $this->label = "Pictures";
        $this->captionLabel = "Captions";
        $this->captionHideIfEmpty = true;
    }

    /** @var string */
    public $captionLabel;

    /** @var boolean */
    public $captionHideIfEmpty;
}
